demo/site/src/pageTypes/Page.tsx

```diff
@@ -37,6 +38,23 @@ export const pageQuery = gql`
     ${topMenuPageTreeNodeFragment}
 `;
 
+export async function loader({
+    client,
+    pageId,
+    contentScope,
+}: {
+    client: GraphQLClient;
+    pageId: string;
+    contentScope: GQLPageTreeNodeScopeInput;
```

Comment:
In Starter we changed this to `scope`, should we do this in Demo as well?

Reply:
done